### DISCUSSION TITLE
Community Partnership proposal for BSides Noida 0x02

### DISCUSSION THREAD

**Discussion by harxh-v** [pydelhi/conference#219] [Community Partnership]
## About BSides Noida:
BSides Noida is a community-driven InfoSec event that aims to cater collaboration, innovation, and knowledge-sharing in the cyber security domain. With over 1,000+ active members and 7,000+ affiliated members globally, we are on a mission to empower enthusiasts and professionals alike.

### By partnering with PyDelhi, we aim to:

> - Bring together like-minded communities to foster mutual growth and learning.
> - Provide your members with priority access and unique opportunities to engage with the cybersecurity world.
> - Showcase PyDelhi as a driving force in supporting tech-focused initiatives.

### What We’re Looking For:

> As a Community Partner, we would love your support in promoting BSides Noida within your network and encouraging participation in our event. Do bring some cool PySwags that no one can beat!

### What’s in It for PyDelhi?

> - Recognition as an official Community Partner across all our socials and shout-out on the event day.
> - Exclusive access to event highlights and potential collaboration opportunities in the future.
> - Special benefits for PyDelhi members, such as priority registrations.

Looking forward to your response!

**Comment by harxh-v** [comment]
Registration Link: https://go.bsidesnoida.in/cm

**Comment by satyaakam** [comment]
@Animesh-Ghosh @Daksh777 @gora what do you guys think about this ?

**Reply by Daksh777** [reply]
We're good to go with this!
